docs/user/openstack/customization.md

```diff
@@ -31,8 +31,8 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 * `trunkSupport` (deprecated string): Whether OpenStack ports can be trunked (`1` for true or `0` for false)
 * `clusterOSImage` (optional string): Either a URL with `http(s)` or `file` scheme to override the default OS image for cluster nodes or an existing Glance image name.
 * `clusterOSImageProperties` (optional list of strings): a list of properties to be added to the installer-uploaded ClusterOSImage in Glance. The default is to not set any properties. `clusterOSImageProperties` is ignored when `clusterOSImage` points to an existing image in Glance.
-* `apiVIP` (optional string): An IP address on the machineNetwork that will be assigned to the API VIP. Be aware that the `10` and `11` of the machineNetwork will be taken by neutron dhcp by default, and wont be available.
-* `ingressVIP` (optional string): An IP address on the machineNetwork that will be assigned to the ingress VIP. Be aware that the `10` and `11` of the machineNetwork will be taken by neutron dhcp by default, and wont be available.
+* `apiVIPs` (optional array of strings): IP address on the machineNetwork that will be assigned to the API VIP. If more than one are set, it must be one IPv4 and one IPv6. Note that the `10` and `11` of the machineNetwork are reserverd for Neutron for DHCP by default.
```

Comment:
Is this note about the `10` and `11` still valid?
I took a look on all the ports for a network with DHCP enabled and couldn't find any port with those addresses.

Reply:
Good point. After digging a bit and asking the Neutron team, it looks like an hallucination on our part.